File: OOPS_LAB/Program_5/account.java
class AccountGen{
    private String HolderName;
    private int AccountNumber;
    private double Balance;

    private int AccountNumGen()
    {
        double acNum = Math.random()*100000000;
        return (int)acNum;
    }

    private double BalanceGen()
    {
        double Balance = Math.random()*10000;
        return Math.round(Balance);
    }

    public AccountGen(String name) {
        this.HolderName=name;
        this.AccountNumber=AccountNumGen();
        this.Balance=BalanceGen();
    }

    public void display()
    {
        System.out.println("Name : "+this.HolderName);
        System.out.println("Account Number "+this.AccountNumber);
        System.out.println("Balance : "+this.Balance);
    }
}

public class account{
    public static void main(String[] args) {
        AccountGen user1 = new AccountGen("Ashfak");
        user1.display();
    }
}

File: OOPS_LAB/Program_5/account.py
import random

class AccountGen:
    def __init__(self, name):
        self.HolderName = name
        self.AccountNumber = self.AccountNumGen()
        self.Balance = self.BalanceGen()

    def AccountNumGen(self):
        acNum = random.random() * 100000000
        return int(acNum)

    def BalanceGen(self):
        balance = random.random() * 10000
        return round(balance)

    def display(self):
        print(f"Name : {self.HolderName}")
        print(f"Account Number : {self.AccountNumber}")
        print(f"Balance : {self.Balance}")

if __name__ == "__main__":
    user1 = AccountGen("Ashfak")
    user1.display()